platforms/documentation/docs/src/docs/userguide/authoring-builds/tasks/custom_tasks.adoc

```diff
@@ -133,29 +157,54 @@ include::{snippetsPath}/base/customExternalTask/groovy/task/src/test/groovy/org/
 [[incremental_tasks]]
 == Incremental tasks
 
-With Gradle, it's very simple to implement a task that is skipped when all of its inputs and outputs are up to date (see <<incremental_build.adoc#incremental_build,Incremental Build>>).
-However, there are times when only a few input files have changed since the last execution, and you'd like to avoid reprocessing all of the unchanged inputs.
+With Gradle, it's straightforward to implement a task that is skipped when all of its inputs and outputs are `UP-TO-DATE` (see <<incremental_build.adoc#incremental_build,Incremental Build>>).
+
+However, there are times when only a few input files have changed since the last execution, and it is best to avoid reprocessing all the unchanged inputs.
 This can be particularly useful for a transformer task that converts input files to output files on a 1:1 basis.
 
-If you'd like to optimize your build so that only out-of-date input files are processed, you can do so with an _incremental task_.
+You can optimize your build so that only out-of-date input files are processed using an *incremental task*.
 
 [[sec:implementing_an_incremental_task]]
 === Implementing an incremental task
 
 For a task to process inputs incrementally, that task must contain an _incremental task action_.
+
 This is a task action method that has a single link:{groovyDslPath}/org.gradle.work.InputChanges.html[InputChanges] parameter.
 That parameter tells Gradle that the action only wants to process the changed inputs.
-In addition, the task needs to declare at least one incremental file input property by using either link:{javadocPath}/org/gradle/work/Incremental.html[@Incremental] or link:{javadocPath}/org/gradle/api/tasks/SkipWhenEmpty.html[@SkipWhenEmpty].
+
+In addition, the task needs to declare at least one incremental file input property by using either link:{javadocPath}/org/gradle/work/Incremental.html[`@Incremental`] or link:{javadocPath}/org/gradle/api/tasks/SkipWhenEmpty.html[`@SkipWhenEmpty`].
+
+[source, groovy]
+.build.gradle
+----
+class IncrementalReverseTask extends DefaultTask {
+    @InputDirectory
```

Comment:
❌ This should come from a snippet that is tested. It lacks the `@Incremental` annotation that is called out in the text above.

Reply:
Fixed.